app/src/main/res/values/strings.xml

```diff
@@ -32,4 +33,11 @@
     <string name="data_sync_search_pokemon">Search a Pokemon</string>
     <string name="data_sync_check_it_out">Check it out!</string>
 
+    <!-- Facebook -->
+    <string name="facebook_login_app_id">329539014322126</string>
```

Comment:
This sounds like it should be in a configuration file or at least en constant somewhere.

Reply:
This is called in the `AndroidManifest.xml`. I put it in here following the Facebook instructions.

Reply:
Yeah, I'm not too comfortable with this one either but my guess was that he followed guidelines. Since this is not a mission critical app, I wouldn't worry too much. It's Facebook's recommendation, if there is a security problem we can troubleshoot it through them.